test/unit/mdc-chips/mdc-chip-set.test.js

```diff
@@ -89,12 +89,18 @@ test('#destroy cleans up child chip components', () => {
 
 test('#initialSyncWithDOM sets up event handlers', () => {
   const {root, mockFoundation} = setupMockFoundationTest();
-
-  domEvents.emit(root, MDCChipFoundation.strings.INTERACTION_EVENT);
-  td.verify(mockFoundation.handleChipInteraction(td.matchers.anything()), {times: 1});
-
-  domEvents.emit(root, MDCChipFoundation.strings.REMOVAL_EVENT);
-  td.verify(mockFoundation.handleChipRemoval(td.matchers.anything()), {times: 1});
+  const evtData = {
+    detail: {chipId: 'chipA'},
+    bubbles: true,
+  };
+  const evt1 = new CustomEvent(MDCChipFoundation.strings.INTERACTION_EVENT, evtData);
```

Comment:
I don't think this is going to fly in IE. (@williamernest can probably give you more details quickly.)

Reply:
Yup, talked to Will - thanks for catching it. Will fix it.